content/blog/knative-eventing-for-observability.md

```diff
@@ -0,0 +1,456 @@
+---
+title: "Using Knative Eventing for Better Observability"
+featured: true
+weight: 1
+description: >
+  Getting a better picture of Kubernetes events
+date: 2021-02-22
+topics:
+  - Kubernetes
+tags:
+  - Kubernetes
+  - API
+  - Development
+  - Knative
+patterns:
+  - Deployment
+# Author(s)
+team:
+  - Tyler Britten
+---
+
+{{< youtube id="9lBaKKe-59E" class="youtube-video-shortcode" >}}
+<div align="center"><i><a href="https://www.youtube.com/watch?v=9lBaKKe-59E&feature=youtu.be">Watch Tyler Walk through these steps on Tanzu.TV.</a></i></div>
+
+
+If you’re using one of the great observability tools out there, you probably already mark your data with important events that may affect it—deployments, configuration changes, code commits, and more. But what about changes Kubernetes makes on its own, like autoscaling events?
```

Comment:
should this be `that may affect it - deployments ...` ?  otherwise it reads as `IT Deployments` to me ?

Reply:
em dash my friend. style guide is no spaces before or after